waku/node/v2/waku_types.nim

```diff
@@ -131,6 +134,9 @@ proc notify*(filters: Filters, msg: WakuMessage) =
       if msg.contentTopic in filter.contentFilter.topics:
         filter.handler(msg.payload)
 
-proc generateRequestId*(): string =
-  for _ in .. 10:
-    add(result, char(rand(int('A') .. int('z'))))
\ No newline at end of file
+proc generateRequestId*(rng: ref BrHmacDrbgContext): string =
```

Comment:
Something I should probably not raise here but in the vacp2p specs, but does it make sense to have this as string and not just seq[byte]?
And if we do want string, shouldn't it be hex encoded?

Reply:
could be hex encoded yeah. Doesn't really matter just needs to be something random. Should I hex encode it? @kdeme

Reply:
I think that makes more sense yes, here is an example: https://github.com/status-im/nim-eth/blob/master/eth/p2p/rlpx_protocols/whisper/whisper_types.nim#L161

Reply:
Because we pass a fixed array now, we can remove the addr/unsafeaddr  and length parts. Pushed a commit for this.